### ISSUE TITLE
Eslint attempts to read config files for removed projects.

### ISSUE THREAD

**Issue by erikh2000** [AtomLinter/linter-eslint#897]
After making an edit to a JS file, this message appears in my developer log console:

/Users/erik.hermansen/.atom/packages/linter/lib/linter-registry.js:159 [Linter] Error running ESLint Error: Cannot read config file: /my-home-dir/**some-old-project-of-mine**/index.js(…)

"some-old-project-of-mine" mentioned above is a project that _used_ to be added to the IDE, but _has been removed_. It is expected that the path that eslint is complaining about would not be present, because that project has been removed. It's unexpected that eslint would continue trying to find files within that removed project folder.

Maybe there is a configuration file someplace that eslint looks at which needs to be updated?

I have the following installed:

- Atom 1.16.0 (MacOS)
- linter 2.1.4
- linter-eslint 8.1.7
- linter-ui-default 1.2.3

Stack trace:

linter-registry.js [sm]:137 [Linter] Error running ESLint Error: Cannot read config file: /my-home-dir/some-old-project-of-mine/index.js(…)

- (anonymous function) @ linter-registry.js [sm]:137
- _tickCallback @ internal/process/next_tick.js:103

**Comment by Arcanemagus** [comment]
Can you post the debug information as requested in the issue template? I'm guessing you configured an override setting file at some point in the settings that is now causing problems.

**Comment by erikh2000** [comment]
Landon, thanks for replying back. Today, I went to get the requested debug information and found that I could no longer reproduce the bug. Possibly, when I deleted my node_modules and reinstalled it cleared out something--not sure. Things I had tried earlier that had no efffect--restarting Atom, disabling/reenabling eslint and linter Atom packages. So this is good news for me using eslint, but bad news for finding the bug. If I see it happen again, I'll try to provide more info.
